package.json

```
@@ -0,0 +1,11 @@
+{
+  "name": "text",
+  "version": "1.0.0",
+  "main": "ppp.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "description": ""
+}
```

1. 엔트리포인트가 main.js
2. import 사용하려면 type이 모듈이어야함
3. start script 추가해주면 `npm run start`로 실행 가능

```suggestion
{
  "name": "text",
  "version": "1.0.0",
  "main": "main.js",
  "scripts": {
    "start": "node main.js",
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "author": "",
  "license": "ISC",
  "description": "",
  "type": "module"
}
```